mne_bids/read.py

```diff
@@ -248,6 +248,10 @@ def _handle_scans_reading(scans_fname, raw, bids_path):
     else:
         acq_times = ['n/a'] * len(fnames)
 
+    # There are three possible extensions for BrainVision
+    if fname.split('.')[1] in ('vhdr', 'eeg', 'vmrk'):
```

Comment:
you don't like my splitext suggestion @teonbrooks ?

Reply:
yes yes, I missed the review when I pushed. needed to do some other maneuvering. let me know what you think

Reply:
also, i just learned great deal about pathlib :)